cartographer_ros/cartographer_ros/node.cc

```diff
@@ -112,6 +113,10 @@ Node::Node(const NodeOptions& node_options, tf2_ros::Buffer* const tf_buffer)
   service_servers_.push_back(node_handle_.advertiseService(
       kWriteAssetsServiceName, &Node::HandleWriteAssets, this));
 
+  trajectory_nodes_list_publisher_ =
```

Comment:
Let's try to group this code a bit more systematically: Move after `submap_list_publisher_` in line 104?